sympy/simplify/fu.py

```diff
@@ -1358,6 +1361,10 @@ def f(rv):
         if not (isinstance(rv, Pow) and isinstance(rv.base, sin)):
             return rv
 
+        e = rv.exp
+        if e % 2 == 1:
+            return TR15(rv.base**(e+1))/rv.base
```

Comment:
```suggestion
            return TR15(rv.base**(e + 1))/rv.base
```

Reply:
In docstring of 15 and 16 could you change power indicator from `*` to `**`, too, please?

Reply:
Done.